<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularioIngreso</class>
 <widget class="QWidget" name="FormularioIngreso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>221</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>INGRESO</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(134, 189, 255);
border-color: rgb(13, 13, 13);</string>
  </property>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>INGRESO DE VEHICULOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/imagenes/ingreso.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>PATENTE        </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lePatente">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);
border: 1px solid #555;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lTiempoIngreso">
      <property name="text">
       <string>FECHA Y HORA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEditIngreso">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);
border: 1px solid #555;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>19</hour>
        <minute>48</minute>
        <second>0</second>
        <year>2018</year>
        <month>9</month>
        <day>21</day>
       </datetime>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::DaySection</enum>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::TimeZone</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>158</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pbCargar">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;
border: 1px solid #555;
border-radius: 9px;
padding: 5px;
background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string>CARGAR</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbVolverIngreso">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333;
border: 1px solid #555;
border-radius: 9px;
padding: 5px;
background-color: rgb(222, 222, 222);</string>
      </property>
      <property name="text">
       <string>VOLVER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lIcono3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recursos.qrc">:/imagenes/imagenes/auto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lWarning">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>146</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="LineaDeTexto" name="leDescripcion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>219</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);
border: 1px solid #555;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DESCRIPCION</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineaDeTexto</class>
   <extends>QLineEdit</extends>
   <header>lineadetexto.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
